scripts/msi-installer-builder/main.go

```diff
@@ -113,6 +121,16 @@ func normalize(preset languagePreset, c *config) (*config, error) {
 	return c, nil
 }
 
+func commitID() (string, error) {
+	cmd := exec.Command("git", "rev-parse", "HEAD")
```

Comment:
The `constants` package already provides this, please use that instead so we don't have multiple implementations of the same thing.

So `constants.RevisionHash`